packages/surveys/src/components/questions/date-question.tsx

```
@@ -134,159 +134,153 @@ export function DateQuestion({
+            {!datePickerOpen && (
+              <button
+                onClick={() => {
+                  setDatePickerOpen(true);
+                }}
+                tabIndex={isCurrent ? 0 : -1}
+                onKeyDown={(e) => {
+                  if (e.key === " ") setDatePickerOpen(true);
+                }}
+                aria-label={selectedDate ? `You have selected ${formattedDate}` : "Select a date"}
+                aria-describedby={errorMessage ? "error-message" : undefined}
+                className="focus:fb-outline-brand fb-bg-input-bg hover:fb-bg-input-bg-selected fb-border-border fb-text-heading fb-rounded-custom fb-relative fb-flex fb-h-[12dvh] fb-w-full fb-cursor-pointer fb-appearance-none fb-items-center fb-justify-center fb-border fb-text-left fb-text-base fb-font-normal">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)